src/pudl_rmi/deprish.py

```diff
@@ -1178,10 +1182,10 @@ def get_plant_pudl_info(pudl_out):
         pudl_out.gens_eia860()
         .assign(count='place_holder')
         .sort_values(['plant_name_eia', 'state'])
-        .groupby(['plant_id_pudl', 'report_date'], as_index=False)
+        .groupby(['plant_id_eia', 'report_date'], as_index=False)
         # Must use .join because x.unique() arrays are not hashable
         .agg(
-            {'plant_id_eia':
+            {'plant_id_pudl':
```

Comment:
Most of the time you're swapping from `plant_id_pudl` to `plant_id_eia` but here you are doing the opposite? Just checking that was intentional

Reply:
this is a good question! but yes in this case it is intentional. I am grouping by the plant_id_eia above so I effectively swapped them (one in the groupby one in the agg).